nemo/collections/asr/parts/submodules/rnnt_beam_decoding.py

```diff
@@ -1319,21 +1367,42 @@ def prefix_search(
                         self.joint.joint(enc_out, hyp_i.dec_out[-1]) / self.softmax_temperature, dim=-1,
                     )
                     logp = logp[0, 0, 0, :]
-
-                    curr_score = hyp_i.score + float(logp[hyp_j.y_sequence[pref_id]])
+                    if self.ngram_lm:
+                        lm_score, next_state = self.compute_ngram_score(
+                            hyp_i.ngram_lm_state, int(hyp_j.y_sequence[pref_id])
+                        )
+                        curr_score = (
+                            hyp_i.score + float(logp[hyp_j.y_sequence[pref_id]]) + self.ngram_lm_alpha * lm_score
+                        )
+                    else:
+                        curr_score = hyp_i.score + float(logp[hyp_j.y_sequence[pref_id]])
```

Comment:
Why not keep the original code and just add if branch after it to add the lm score ?

Reply:
Sounds reasonable, thanks)